src/main/java/randoop/reflection/OperationModel.java

```diff
@@ -106,28 +107,30 @@ private OperationModel() {
   /**
    * Factory method to construct an operation model for a particular set of classes
    *
-   * @param visibility the {@link randoop.reflection.VisibilityPredicate} to test accessibility of
-   *     classes and class members
+   * @param visibility the {@link VisibilityPredicate} to test accessibility of classes and class
+   *     members
    * @param reflectionPredicate the reflection predicate to determine which classes and class
    *     members are used
+   * @param omitPredicate the patterns for operations that should be omitted
    * @param classnames the names of classes under test
    * @param coveredClassnames the names of classes to be tested by covered class heuristic
    * @param methodSignatures the signatures of methods to be added to the model
    * @param errorHandler the handler for bad file name errors
    * @param literalsFileList the list of literals file names
-   * @return the operation model for the parameters
-   * @throws OperationParseException if a method signature is ill-formed
+   * @return the {@link OperationModel} constructed with the given arguments
+   * @throws SignatureParseException if a method signature is ill-formed
    * @throws NoSuchMethodException if an attempt is made to load a non-existent method
    */
   public static OperationModel createModel(
       VisibilityPredicate visibility,
       ReflectionPredicate reflectionPredicate,
+      OmitMethodsPredicate omitPredicate,
```

Comment:
Why does OperationModel.createModel take an OmitMethodsPredicate rather than a list of patterns?
OmitMethodsPredicate is only ever created and immediately passed into OperationModel.createModel.
How about making OperationModel.createModel take as input a List<Pattern> instead?  That can move the creation of the OmitMethodsPredicate into OperationModel, so it will appear once in the source code instead of many times.